contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp

```diff
@@ -180,7 +180,7 @@ class TsqlUnsupportedFeatureHandlerImpl : public TsqlUnsupportedFeatureHandler
 		antlrcpp::Any visitFunction_call(TSqlParser::Function_callContext *ctx) override;
 		antlrcpp::Any visitAggregate_windowed_function(TSqlParser::Aggregate_windowed_functionContext *ctx) override;
 		antlrcpp::Any visitRowset_function(TSqlParser::Rowset_functionContext *ctx) override {
-			if (!ctx->open_json() && !ctx->open_query()) {
+			if (!ctx->open_json() && (!pltsql_linked_servers_enabled || !ctx->open_query())) {
```

Comment:
You should add a test to make sure that when GUC is disabled, all features are blocked as expected